docs/index.rst

```diff
@@ -4,29 +4,34 @@ Read the Docs: Documentation Simplified
 .. meta::
    :description lang=en: Automate building, versioning, and hosting of your technical documentation continuously on Read the Docs.
 
-
 `Read the Docs`_ simplifies software documentation
-by automating building, versioning, and hosting of your docs for you.
+by building, versioning, and hosting of your docs, automatically.
 Think of it as *Continuous Documentation*.
 
-Never out of sync
+Never out of sync |:arrows_counterclockwise:|
```

Comment:
:heart:

Reply:
Emojis are really good here.

BTW, I don't like how this renders in the new version of the theme, but I suppose that's a bug.